service/src/main/java/tds/session/services/impl/SessionServiceImpl.java

```diff
@@ -23,4 +23,11 @@ public SessionServiceImpl(SessionRepository sessionRepository) {
     public Optional<Session> getSessionById(UUID id) {
         return sessionRepository.getSessionById(id);
     }
+
+    @Override
+    public void pause(final UUID sessionId, final String newStatus) {
+        // TODO:  Throw an exception if attempting to pause a Session that is already paused?
```

Comment:
... I could also see returning a 404 if we can't find a Session for the ID that was passed in.